browser/src/slideshow/PresenterConsole.js

```diff
@@ -900,13 +900,13 @@ class PresenterConsole {
 
 		if (this._notes) {
 			let notes = this._presenter.getNotes(e.slide);
-			if (notes) elem = this._notes.querySelector('#notes');
-			if (elem) {
-				elem.innerText =
-					notes && notes.toLowerCase() !== 'click to add notes'.toLowerCase()
-						? notes
-						: _('No Notes');
-			}
+			let notesContentElem = this._notes.querySelector('#notes');
+			notesContentElem.innerText = _('No Notes');
+			if (
+				notes &&
+				notes.toLowerCase() !== _('click to add notes').toLowerCase()
```

Comment:
this string is not set anywhere: "click to add notes"
does it come from core? we shouldn't compare translated versions as core and online might be different...

Reply:
@eszkadev so should we just do normal string comparison ?

Reply:
https://github.com/CollaboraOnline/online/pull/10521#issuecomment-2484080473

Reply:
we should use some not fragile approach, which will work if core will change translation